deepmd/pt/infer/deep_eval.py

```diff
@@ -178,7 +178,7 @@ def get_sel_type(self) -> List[int]:
 
     def get_numb_dos(self) -> int:
         """Get the number of DOS."""
-        return 0
+        return self.dp.model["Default"].get_fitting_net().dim_out
```

Comment:
Are you sure this can be visited from a jit model?

Reply:
When we freeze a model to `model.pth` and load it with DeepEval, it will visit this method.

Reply:
You may need to export `get_fitting_net` explicitly. (currently, it's implicitly exported, which is not safe)

Reply:
export in `DOSModel`